internal/server_control/server_control_test.go

```diff
@@ -0,0 +1,77 @@
+package server_control
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/Minecraft-Unified-Hub-Team/ServerControl/internal/api"
+)
+
+func TestMinecraftConfigToMap(t *testing.T) {
```

Comment:
I think you should use common error formating here (because if error happens you don't write part of stack in it). Also use errors.New()